nvm.sh

```diff
@@ -1044,6 +1044,17 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  if [ "_$NVM_OS" = "_linux" ]; then
+    MAKE_JOBS="$(nproc)"
+  elif [ "_$NVM_OS" = "_freebsd" ] || [ "_$NVM_OS" = "_darwin" ]; then
+    MAKE_JOBS="$(sysctl -n hw.ncpu)"
+  elif [ "_$NVM_OS" = "_sunos" ]; then
+    MAKE_JOBS="$(psrinfo | wc -l)"
+  else
+    MAKE_JOBS="1"
```

Comment:
It will be very special case I think, but if you want, I can modify it to:
```bash
  if [ "_$NVM_OS" = "_linux" ]; then
    MAKE_JOBS="$(nproc)"
  elif [ "_$NVM_OS" = "_freebsd" ] || [ "_$NVM_OS" = "_darwin" ]; then
    MAKE_JOBS="$(sysctl -n hw.ncpu)"
  elif [ "_$NVM_OS" = "_sunos" ]; then
    MAKE_JOBS="$(psrinfo | wc -l)"
  fi
  if [ -z "$MAKE_JOBS" ]; then
    MAKE_JOBS="1"
  fi
```